config-linux.md

```diff
@@ -531,12 +529,9 @@ The following parameters can be specified to set up seccomp:
     * `SCMP_ARCH_S390X`
     * `SCMP_ARCH_PARISC`
     * `SCMP_ARCH_PARISC64`
-
 * **`syscalls`** *(array of objects, OPTIONAL)* - match a syscall in seccomp.
-
     While this property is OPTIONAL, some values of `defaultAction` are not useful without `syscalls` entries.
     For example, if `defaultAction` is `SCMP_ACT_KILL` and `syscalls` is empty or unset, the kernel will kill the container process on its first syscall.
-
```

Comment:
I think this “you'll probably want to set this” text is worth keeping in its own paragraph.  But I don't feel too strongly about it, and am ok with all of this getting squashed into a single paragraph like you're doing here.